test/integration/ingress_test.go

```diff
@@ -38,15 +39,15 @@ var ingressClassMutex = sync.Mutex{}
 func TestIngressEssentials(t *testing.T) {
 	t.Parallel()
 
-	t.Log("locking IngressClass management")
+	t.Log("locking consts.IngressClass management")
```

Comment:
Renaming in this file went wrong 😛 (`consts.IngressClass` in log messages).

Reply:
👍